AndreiFamishkin/01.html/task1.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Пример страницы</TITLE>
+	</HEAD>
+<!-- начало документа -->
+	<BODY> 
+<!-- Заголовок страницы -->
+		<H1>Это заголовок</H1>
+			<P><DEL>Добрый день, <EM>Уважаемые слушатели</EM></DEL>!
+			Сегодня мы с вами будем рассматривать структуру стандартного <STRONG>HTML</STRONG> файла.</P>
+			<P>Далее мы будем использовать произвольный текст для наполнения контентом. 
+			Не стоит вдумываться в его содержание.</P>
+			<P>«Си́мпсоны» (англ. The Simpsons) — самый длинный мультсериал в истории американского телевидения,
+			длящийся 25 сезонов. Первая мини-серия «Good night» была показана в «Шоу Трейси Ульман» 19 апреля 1987 года.
+<!-- начало комментария -->			
```

Comment:
In lines 17 and 30 you have two comments.
It should be one comment started from line 17 and will ended on line 30